File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{   int height;
    do
    {
        height = get_int("Height: ");
    }
    while(height <  1 || height > 8);

    for(int row = 0; row< height; row++)
    {
        for(int space = height - row - 1; space > 0; space--)
        {
            printf(" ");
        }
        for(int k = 0; k < row + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# Loop until condition met
while True:
    height = get_int("Height: ")

    if height >= 1 and height <= 8:
        break

hashes = 1

# Print blocks
for i in range(height, 0, -1):
    print(" " * (i - 1), end="")
    print("#" * hashes, end="")

    print("  ", end="")

    print("#" * hashes)

    hashes += 1